### ISSUE TITLE
autocomplete attribute has valid value 73f2c2 - applicability for aria-hidden=true

### ISSUE THREAD

**Issue by dd8** [act-rules/act-rules.github.io#1677]
The rules doesn't apply if the following is true

> hidden: the element is not visible, and **not included in the accessibility tree**; or

So this means the following doesn't fail the rule:

```
<input type='text' name='your-name' autocomplete='badvalue' aria-hidden='true'>
```

However the benefits section of Understanding 1.3.5: Identify Input Purpose mentions people who interact with input fields using browser autocomplete without any assistive tech, so aria-hidden has no effect:
https://www.w3.org/WAI/WCAG21/Understanding/identify-input-purpose.html#benefits

And the prototype assistive tech tools mentioned by John Foliot injects CSS styling on the DOM , and doesn't use the accessibility tree (so again aria-hidden has no effect). I think it works something like this:

```
 input[ autocomplete = 'cc-number' ]:after { content: url(credit-card-icon.png'); }
```

https://www.slideshare.net/johnfoliot/135-more-than-autocomplete

**Comment by dd8** [comment]
Doh - missed the fact the wording is "and" and not "or"
